### PR TITLE
fix reset cypher

### PR DESCRIPTION
**Pull request by martinohanlon**
reset.cypher in GDS Community Detection deletes all Person nodes when it should load them

### CODE REVIEW

**Comment by martinohanlon** [comment]
@adam-cowley converted to one query.

File: asciidoc/courses/gds-community-detection/modules/2-preprocess/lessons/3-c-categorical/reset.cypher [hunk 1]
```diff
@@ -1,10 +1,17 @@
-// Delete All nodes
-MATCH (p:Person) DETACH DELETE p;
+// reload data
+:auto LOAD CSV
```
Comment (adam-cowley) [inline asciidoc/courses/gds-community-detection/modules/2-preprocess/lessons/3-c-categorical/reset.cypher:2]: `:auto` is a command to tell Browser to run the query in an auto-commit transaction. 

Thinking about it, you should be able to do it without the subquery:

```
LOAD CSV
WITH HEADERS
FROM "https://raw.githubusercontent.com/tomasonjo/blog-datasets/main/survey/responses.csv" AS row

CREATE (p:Person)
SET p = row
WITH p
UNWIND keys(p) AS key
WITH p, key, toFloat(p[key]) AS value
WHERE value IS NOT null
CALL apoc.create.setProperty(p, key, value) YIELD node
RETURN DISTINCT 'done' AS result
```

**Comment by adam-cowley** [comment]
Could you also update `asciidoc/courses/gds-community-detection/modules/2-preprocess/lessons/4-dimensionality/reset.cypher` while you have the PR open, please?

**Comment by martinohanlon** [comment]
Done

**Pull request merged**